Makefile

```diff
@@ -1,76 +1,93 @@
 # Makefile for building litmus-e2e
+TESTPATH ?= /home/udit/go/src/github.com/litmuschaos/litmus-e2e/tests/
```

Comment:
why the name here is udit?

Reply:
`/home/udit`
is the home dir of the Landing Machine. (target cluster).
Couldn't access the Landing Machine variables in Makefile otherwise, I would have replaced it with` ${HOME}`